packages/connect-core/src/entities/Organization.ts

```diff
@@ -134,4 +136,10 @@ export default class Organization {
       this.connection.ethersProvider
     )
   }
+
+  //////// DESCRIPTIONS /////////
+
+  async describeScript(script: string): Promise<ForwardingPathDescription> {
```

Comment:
I decided to expose the `describeScript` utility to the organization as we already have all the contextual information we need and user and app connectors can leverage directly.